src/ruby/install.sh

```diff
@@ -7,26 +7,21 @@
 # Docs: https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/docs/ruby.md
 # Maintainer: The VS Code and Codespaces Teams
 
-RUBY_VERSION="${VERSION:-"latest"}"
-
-USERNAME="${USERNAME:-"${_REMOTE_USER:-"automatic"}"}"
-UPDATE_RC="${UPDATE_RC:-"true"}"
-INSTALL_RUBY_TOOLS="${INSTALL_RUBY_TOOLS:-"true"}"
+export RUBY_VERSION="${VERSION:-"latest"}"
 
 # Comma-separated list of ruby versions to be installed (with rvm)
 # alongside RUBY_VERSION, but not set as default.
 ADDITIONAL_VERSIONS="${ADDITIONALVERSIONS:-""}"
 
-# Note: ruby-debug-ide will install the right version of debase if missing and
-# installing debase directly fails on Ruby 3.1.0 as of 1/7/2022, so omitting.
-# installing ruby-debug-ide on debian fails, so omitting.
-DEFAULT_GEMS="rake"
+GPG_KEY_SERVERS="keyserver hkps://keys.openpgp.org
+keyserver hkp://keyserver.pgp.com
+keyserver hkp://pgp.mit.edu
+keyserver hkp://ipv4.pool.sks-keyservers.net
+keyserver hkp://keyserver.ubuntu.com
+keyserver hkp://keyserver.ubuntu.com:80"
```

Comment:
I added the keyservers recommended in the RVM documentation in addition to the ones previously included.